### PR TITLE
Upgrade test: wait for CSV

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/upgrade-test.sh [hunk 1]
```diff
@@ -210,6 +210,11 @@ echo "----- HCO deployOVS annotation and OVS state in CNAO CR before the upgrade
 PREVIOUS_OVS_ANNOTATION=$(${CMD} get ${HCO_KIND} ${HCO_RESOURCE_NAME} -n ${HCO_NAMESPACE} -o jsonpath='{.metadata.annotations.deployOVS}')
 PREVIOUS_OVS_STATE=$(${CMD} get networkaddonsconfigs cluster -o jsonpath='{.spec.ovs}')
 
+# Before starting the upgrade, make sure the CSV is installed properly.
+Msg "Read the CSV to make sure the deployment is done"
+./hack/retry.sh 30 10 "${CMD} get ClusterServiceVersion  -n ${HCO_NAMESPACE} kubevirt-hyperconverged-operator.v${INITIAL_CHANNEL} -o jsonpath='{ .status.phase }') | grep 'Succeeded'"
```
Comment (orenc1) [inline hack/upgrade-test.sh:215]: syntax error - please remove the ')'  at the end.

**Review by tiraboschi — APPROVED** [review]

**Pull request merged**